src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Figure implements Area, Output{
+    double a, b;
+
+    @Override
+    public double findArea() {
+        return a * b;
+    }
+
+    @Override
+    public String printAllInformation() {
```

Comment:
The method 'printAllInformation' should return some actual information about the figure. Returning 'null' is not informative and could lead to a NullPointerException if the method's return value is used without checking.